tests/test_build_latex.py

```diff
@@ -1383,3 +1383,19 @@ def test_default_latex_documents():
     expected = [('index', 'stasi.tex', 'STASI™ Documentation',
                  r"Wolfgang Schäuble \& G'Beckstein.\@{}", 'manual')]
     assert default_latex_documents(config) == expected
+
+
+@skip_if_requested
+@skip_if_stylefiles_notfound
+@pytest.mark.sphinx('latex', testroot='latex-includegraphics',
+    confoverrides={'latex_documents': [
+                       ('index', 'python.tex', 'Test PDF build',
```

Comment:
on 1.8 branch I would use here SphinxTests

Reply:
Is this option still needed on master branch?

Reply:
No, it works fine without it.